sympy/simplify/tests/test_trigsimp.py

```diff
@@ -351,8 +351,9 @@ def test_issue_2827_trigsimp_methods():
     # all methods should work with Basic expressions even if they
     # aren't Expr
     M = Matrix.eye(1)
+    # NOTE: "groebner" is currently not tested as it raises an exception:
     assert all(trigsimp(M, method=m) == M for m in
-        'fu matching groebner old'.split())
```

Comment:
@asmeurer had to disable _groebner_, as the matrix is now a non-commutative polynomial, and apparently they are not supported.

Reply:
Shouldn't `trigsimp` on a matrix just apply to the elements of the matrix?